app/javascript/components/pages/Home.js

```diff
@@ -2,7 +2,36 @@ import React from 'react'
 
 const Home = () => {
   return (
-    <div>Home</div>
+    <>
```

Comment:
Just a note - you don't need a fragment and a div. As long as one thing is wrapping all other tags.